server/schedulers/hot_region_config.go

```diff
@@ -37,6 +37,9 @@ const (
 	KeyPriority = "key"
 	// QueryPriority indicates hot-region-scheduler prefer query dim
 	QueryPriority = "qps"
+
+	// Scheduling has a bigger impact on TiFlash, so it needs to be corrected in configuration items
+	tiflashToleranceRatioCorrection = 0.1
```

Comment:
The number is magic, does it have a big impact?

Reply:
Scheduling has more impact on TiFlash than TiKV, so bigger tolerance is required.